src/pages/library/DataBrowserToggler.js

```diff
@@ -142,6 +127,10 @@ export const DataBrowserPreviewToggler = ({ checked }) => {
     [div({ style: { fontWeight: 600, fontSize: 18 } },
       'Thank you for helping us improve the Data Catalog experience!')]
     )
-  ])
+  ]), !catalogShowing ? div({
+    style: {
+      padding: '15px 15px', wordWrap: 'break-word', width: 300, lineHeight: 1.5
```

Comment:
Why limit the width of this and make the text wrap?
![Screen Shot 2022-07-05 at 3 41 32 PM](https://user-images.githubusercontent.com/1156625/177403335-ab130fbf-2ba8-4cde-ae8f-9ba815680069.png)

Reply:
just to match our UI mock
https://projects.invisionapp.com/d/main?origin=v7#/console/22047689/467561712/preview